components/switch/overview.md

```diff
@@ -37,116 +37,78 @@ The <a href = "https://www.telerik.com/blazor-ui/switch" target="_blank">Blazor
 
 ![Telerik Switch Component](images/swtich-first-look.gif)
 
-## Features
+## Labels
 
-The Switch provides the following features:
+The Switch has dedicated `On` and `Off` labels serving as text representation of the component value. [Read more about the Switch labels....]({%slug switch-labels%})
 
-* `Class` - the CSS class that will be rendered on the main wrapping element of the Switch. Use it to [customize the Switch background color and other styles]({%slug switch-kb-change-background-color%})
+## Events
 
-* `Enabled` - whether the component is enabled.
+The Blazor Switch generates events that you can handle and further customize its behavior. [Read more about the Switch events....]({%slug switch-events%})
 
-* `Id` - renders as the `id` attribute on the wrapping `<span>` element of the component.
+## Validation
 
-* `Width` - the width of the component.
+You can ensure that the component value is acceptable by using the built-in validation. [Read more about input validation....]({%slug common-features/input-validation%}) 
 
-* `TabIndex` - the `tabindex` attribute rendered on the Switch.
+## Switch Parameters
 
-* `Value` and `bind-Value` -  accept `bool` and `bool?` data types.
+The Switch provides the following features to allow further customization of its behavior:
 
-* `OnLabel` - the label of the component when the `Value` is `true`.
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
-* `OffLabel` - the label of the component when the `Value` is `false`.
+| Attribute | Type and Default Value | Description |
+|----------|----------|----------|
+| `Enabled` | `bool` | whether the component is enabled.
+| `Id` | `string` | renders as the `id` attribute on the wrapping `<span>` element of the component.
+| `TabIndex` | `int` | the `tabindex` attribute rendered on the Switch.
+| `Value` and `bind-Value` | `bool` or `bool?`| the value of the Switch.
```

Comment:
@bind-Value is not a separate parameter that requires mentioning. "@bind" is a Blazor directive. We can specify that it supports two-way binding in the description.